modules/apps/object/object-test/src/testFunctional/tests/Object.testcase

```diff
@@ -300,6 +300,47 @@ definition {
 			value1 = "Input does not match");
 	}
 
+	@description = "Verify it's possible to sort object entries by API"
+	@priority = "4"
+	test CanSortObjectEntriesViaAPI {
+		task ("Create an object") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 158616",
+				objectName = "CustomObject158616",
+				pluralLabelName = "Custom Objects 158616");
+		}
+
+		task ("Add Field to the object") {
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject158616");
+		}
+
+		task ("Publish object") {
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject158616");
+		}
+
+		task ("Add entries to the object") {
+			for (var letter : list "C,Z,A") {
+				ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+					fieldName = "customObjectField",
+					objectName = "CustomObject158616",
+					value = "Entry${letter}");
+			}
+		}
+
+		task ("Assert sorted order of elements") {
+			ObjectAdmin.assertObjectSortedEntry(
+				expectedValue = "EntryA,EntryC,EntryZ",
+				objectName = "CustomObject158616",
+				sortString = "customObjectField");
+		}
+	}
```

Comment:
Can we  use Given When Then instead?

Given: An custom object with a text field
- Add custom object with field and publish

When: Three object entries are created and sorted using the API
- Add object entries
- Hit the endpoint and store the result in a var

Then: The object entries are returned in alphabetical order

- Assertion only